pgdrive/scene_creator/vehicle/base_vehicle.py

```diff
@@ -478,14 +478,11 @@ def lane_distance_to(self, vehicle, lane: AbstractLane = None) -> float:
 
     def _add_chassis(self, pg_physics_world: PGPhysicsWorld):
         para = self.get_config()
+        self.LENGTH = self.vehicle_config["vehicle_length"]
+        self.WIDTH = self.vehicle_config["vehicle_width"]
```

Comment:
Could we use self.length, self.width here?

By the way, they seem to be class variables. Would changing them here affects other vehicles??

Reply:
Or could we just remove them and ask self.vehicle_config for all times?? 

Think about the case, if we change the `vehicle_config[vehicle_length]` during runtime, what would happen?

Reply:
1. it occurs when you assign a variable to cls.LENGTH, will not affect other instances if you assign a value to self.LENGTH. But I will use length/width instead of LENGTH/WIDTH
2. It is nonsense since its value is used to create rigid body for chassis. Modifying its value in runtime will bring some errors in spawn points calculation and traffic vehicle decision

Reply:
Good

Reply:
humm, it can not be renamed, since Traffic Vehicle will query BaseVehicle.Length